### PR TITLE
Feature/happy front end

### PR DESCRIPTION
**Pull request by flexbox**

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 activate :bower
+activate :livereload
 
-set :css_dir,     'stylesheets'
-set :images_dir,  'images'
-set :js_dir,      'javascripts'
+set :js_dir,     'assets/javascripts'
```
Comment (cveneziani) [inline config.rb]: Don't you like my alphabetical sorting? :grinning:

File: config.rb [hunk 2]
```diff
@@ -18,6 +19,11 @@
     config.sass_options = { debug_info:     false }
     config.sass_options = { line_comments:  false }
   end
+
+  activate :minify_html, remove_input_attributes: false
```
Comment (cveneziani) [inline config.rb]: sorting [hysteria]

**Comment by cveneziani** [comment]
Review done.

**Comment by flexbox** [comment]
Fix Done, 
If you agree you can proceed to merge

**Comment by cveneziani** [comment]
Thanks!

**Pull request merged**